pkg/net/cidrs/aws/internal/ranges2go/parse_test.go

```diff
@@ -77,3 +77,56 @@ func TestParseIPRangesJSON(t *testing.T) {
 		t.Fatal("expected error parsing garbage data but got none")
 	}
 }
+
+func TestRegionsToPrefixesFromData(t *testing.T) {
+	t.Run("bad IPv4 prefixes", func(t *testing.T) {
```

Comment:
why not test valid prefixes too?  ... and use a table test, only expectError is tested

Reply:
there's other tests that cover this method, but we do not cover the error cases in those tests.

Reply:
table test seems excessive for N=2.

Reply:
ack